### ISSUE TITLE
fail to upgrade to 10.6.2

### ISSUE THREAD

**Issue by brimwats** [YunoHost-Apps/jellyfin_ynh#30]
see log here: https://paste.yunohost.org/raw/inozokejox

**Comment by lentemoore** [comment]
I have the same issue, here is the error:

` /opt/yunohost/jellyfin/jellyfin-server.deb

Errors were encountered while processing:

deconfiguration is not permitted (--auto-deconfigure might help)

installing jellyfin-server would break jellyfin, and

dpkg: error processing archive /opt/yunohost/jellyfin/jellyfin-server.deb (--install):

jellyfin (version 10.5.5-1) is present and installed.

jellyfin-server breaks jellyfin (<< 10.6.0)

dpkg: regarding .../jellyfin/jellyfin-server.deb containing jellyfin-server:`

**Comment by liberodark** [comment]
Yeah is normal you need to reinstall have breaking on 10.5.x and 10.6.x

**Comment by brimwats** [comment]
> Yeah is normal you need to reinstall have breaking on 10.5.x and 10.6.x
> Other solution is to force install manually.

thanks! will it delete all data if I uninstall it through the web interface? or forcing it from the CLI works okay?

**Comment by lentemoore** [comment]
Forcing install from CLI doesn't work, I can't override the "jellyfin is installed" error. From the webapp, an uninstall & reinstall also fails, again with the same error:

```
2020-08-25 10:03:10,756: WARNING - dpkg: regarding .../jellyfin/jellyfin-server.deb containing jellyfin-server:
2020-08-25 10:03:10,757: DEBUG - + local exit_code=1
2020-08-25 10:03:10,759: WARNING -  jellyfin-server breaks jellyfin (<< 10.6.0)
2020-08-25 10:03:10,761: DEBUG - + '[' 1 -eq 0 ']'
2020-08-25 10:03:10,762: WARNING -   jellyfin (version 10.5.5-1) is present and installed.
2020-08-25 10:03:10,764: DEBUG - + trap '' EXIT
2020-08-25 10:03:10,764: WARNING - 
2020-08-25 10:03:10,766: DEBUG - + set +o errexit
2020-08-25 10:03:10,767: WARNING - dpkg: error processing archive /opt/yunohost/jellyfin/jellyfin-server.deb (--install):
2020-08-25 10:03:10,769: DEBUG - + set +o nounset
2020-08-25 10:03:10,770: WARNING -  installing jellyfin-server would break jellyfin, and
2020-08-25 10:03:10,773: DEBUG - + sleep 0.5
2020-08-25 10:03:10,774: WARNING -  deconfiguration is not permitted (--auto-deconfigure might help)
2020-08-25 10:03:10,775: WARNING - Errors were encountered while processing:
2020-08-25 10:03:10,777: WARNING -  /opt/yunohost/jellyfin/jellyfin-server.deb
2020-08-25 10:03:11,281: DEBUG - + type -t ynh_clean_setup
2020-08-25 10:03:11,282: DEBUG - + ynh_clean_setup
2020-08-25 10:03:11,284: DEBUG - + true
2020-08-25 10:03:11,285: DEBUG - + exit 1
2020-08-25 10:03:11,338: ERROR - Could not install jellyfin: An error occurred inside the app installation script
```

**Comment by liberodark** [comment]
yes is not simple to switch on new version only solution for switch is to use apt dont use dpkg -i

**Comment by brimwats** [comment]
> yes is not simple to switch on new version only solution :
> Use a specific version !
> 
> `sudo yunohost app upgrade jellyfin -u https://github.com/YunoHost-Apps/jellyfin_ynh/tree/10.6.0`
> 
> This version have been make for upgrade old version to new version (with this process you don't lost your data).
> After that you need to use :
> 
> `sudo yunohost app upgrade jellyfin -u https://github.com/YunoHost-Apps/jellyfin_ynh/`

Unfortunately, this does not work. Here are the error logs https://paste.yunohost.org/raw/taqocejuse

**Comment by brimwats** [comment]
My jellyfin installation is now giving me a Error 502 Bad Gateway

**Comment by brimwats** [comment]
> My jellyfin installation is now giving me a Error 502 Bad Gateway

Okay so running 

> sudo yunohost app upgrade jellyfin -u https://github.com/YunoHost-Apps/jellyfin_ynh/tree/10.6.0 

after running 

> sudo yunohost app upgrade jellyfin -u https://github.com/YunoHost-Apps/jellyfin_ynh/



seems to fix it and I am on 10.6.0
